pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2937,6 +2937,7 @@ type PriorityClassNamesSpec map[KeyType]string
 // +nullable
 type StorageClassDeviceSet struct {
 	// Name is a unique identifier for the set
+	// +kubebuilder:validation:MaxLength=63
```

Comment:
Is 63 the length that actually causes the issue? Or is it actually shorter because the name of the device set is concatenated with something else to create the resource name?

Reply:
That quite a lot of characters added. Are the asterisks also added? I'm a bit confused. I don't think asterisks are allowed characters.

Reply:
Actually asterisks are placeholder here for the real names